C#/C#.sublime-syntax

```diff
@@ -1377,6 +1346,49 @@ contexts:
     - match: (?=\}|\)|>|\]|,|{{reserved}})
       pop: true
 
+  tuple:
+    - meta_scope: meta.group.tuple.cs
```

Comment:
Would we probably want to use `meta.sequence.tuple` and corresponding punctuations to line up with python?

Reply:
I agree. I took a look into this, and it's a complicated change. Maybe I can do it as part of a separate PR please? :)

Reply:
Sure.